### ISSUE TITLE
[Question] Is Kodeine dead for good?

### ISSUE THREAD

**Issue by rahaaatul** [theothertored/kodeine#1]

**Comment by theothertored** [comment]
TL;DR: yes

Rant:

If by "is kodeine dead" you mean "are you still working on this", the answer is no, I'm not. I've tried to design a system for mimicking more of what Kustom does, but over time I've realized the scope of the project is huge, the audience very niche, and I have no interest in sinking what would likely end up being tens or even hundreds of hours into it. I've become much busier and I realized that my time and brain power is in very limited supply and I need to be smart about what I choose to invest those resources into.

I've gravitated away from customization because it doesn't make me more productive (weighing time investment into customizing vs reward of doing a thing very slightly faster) and I don't really find enjoyment in it anymore. I don't have the brain capacity to devote a lot of time into design and then implementation. Building things in the Kustom editor is unreasonably inefficient which just annoys me these days. I've started another project called [Bridge Launcher](https://github.com/bridgelauncher) which lets you build your own launcher using HTML + CSS + JS, and even with that I'm struggling to find any interest in customization, perhaps because I've made the barrier to entry pretty high - it takes a lot of effort to make a launcher from scratch, even when you don't have to deal with Android development hell.

The code for kodeine remains open source, anyone that is interested in investing that time is welcome to fork the project and do literally whatever with it. If someone were to try and do something with it I'm willing to help by answering questions, but don't expect further development from me.
